
{'date': 'Thu May 02 2019 10:20:52.658 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 133
    o_universo = Universo()
  module <module> line 123
    estacao = Estacao(self.universo)
  module <module> line 92
    self.estacao = Elemento(ESTACAO, style=dict(left=left, top=top, width="200px"), vai=self.pega)
NameError: name 'left' is not defined
'''},
{'date': 'Thu May 02 2019 10:30:37.410 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 144
    o_universo = Universo()
  module <module> line 137
    foguete = foguete(self.universo, left="800px", top="100px")
UnboundLocalError: local variable 'foguete' referenced before assignment
'''},
{'date': 'Thu May 02 2019 11:30:43.713 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 163
    o_universo = Universo()
  module <module> line 156
    pessoaa = pessoa(self.universo, left="200px", top="50px")
NameError: name 'pessoa' is not defined
'''},
{'date': 'Thu May 02 2019 11:31:00.850 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 163
    o_universo = Universo()
  module <module> line 156
    pessoaa = Pessoa(self.universo, left="200px", top="50px")
NameError: name 'Pessoa' is not defined
'''},
{'date': 'Thu May 02 2019 11:31:39.601 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 163
    o_universo = Universo()
  module <module> line 156
    pessoa = Pessoa(self.universo, left="200px", top="50px")
NameError: name 'Pessoa' is not defined
'''},
{'date': 'Thu May 02 2019 11:32:00.427 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 163
    o_universo = Universo()
  module <module> line 156
    pessoa = Pessoa(self.universo, left="200px", top="30px")
NameError: name 'Pessoa' is not defined
'''},
{'date': 'Thu May 02 2019 11:33:17.669 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 163
    o_universo = Universo()
  module <module> line 156
    pessoa = PESSOA(self.universo, left="200px", top="30px")
TypeError: 'str' object is not callable
'''},
{'date': 'Thu Jun 06 2019 09:10:32.667 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  PESSOAS = [IMGUR.format(icone) for icone in "H0sLGRO.png rJXklVA.png WjXPkDj.png 31bHmRB.png kSEbjOG.png " 
                                                                                                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Jun 06 2019 09:12:48.873 GMt-0300 (Brasilia Standard Time) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  PESSOAS = [IMGUR.format(icone) for icone in "H0sLGRO.png rJXklVA.png WjXPkDj.png 31bHmRB.png kSEbjOG.png " 
                                                                                                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Jun 06 2019 09:14:38.587 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  PESSOAS = [IMGUR.format(icone) for icone in "H0sLGRO.png rJXklVA.png WjXPkDj.png 31bHmRB.png kSEbjOG.png " 
                                                                                                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Jun 06 2019 09:14:44.517 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  PESSOAS = [IMGUR.format(icone) for icone in "H0sLGRO.png rJXklVA.png WjXPkDj.png 31bHmRB.png kSEbjOG.png " 
                                                                                                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Jun 06 2019 09:14:49.436 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  PESSOAS = [IMGUR.format(icone) for icone in "H0sLGRO.png rJXklVA.png WjXPkDj.png 31bHmRB.png kSEbjOG.png " 
                                                                                                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Jun 06 2019 09:15:11.988 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  PESSOAS = [IMGUR.format(icone) for icone in "H0sLGRO.png rJXklVA.png WjXPkDj.png 31bHmRB.png kSEbjOG.png " 
                                                                                                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Jun 06 2019 09:55:45.451 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 167
    o_universo = Universo()
  module <module> line 156
    estacao = Estacao(self.universo_Estacao.esquerda , left="80px", top="250px", icone=PESSOA)
AttributeError: 'Universo' object has no attribute 'universo_Estacao'
'''},
{'date': 'Thu Jun 06 2019 09:59:33.413 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 155
  estacao = Estacao(self.universo, left="250px", top="80px", icone=ANIMAL))
                                                                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Jun 06 2019 10:06:08.758 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 166
    o_universo = Universo()
  module <module> line 154
    terra = Terra(self.universo, left="200px", top="280px")
TypeError: __init__() got an unexpected keyword argument 'left'
'''},
{'date': 'Thu Jun 06 2019 10:08:33.376 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 166
    o_universo = Universo()
  module <module> line 155
    estacao = Estacao(self.universo, left="250px", width="400px", top="300px", icone=PESSOA)
TypeError: __init__() got an unexpected keyword argument 'width'
'''},
{'date': 'Thu Jun 06 2019 10:09:00.296 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 166
    o_universo = Universo()
  module <module> line 155
    estacao = Estacao(self.universo, left="250px", top="300px", width="400px", icone=PESSOA)
TypeError: __init__() got an unexpected keyword argument 'width'
'''},
{'date': 'Thu Jun 06 2019 10:12:31.826 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 166
    o_universo = Universo()
  module <module> line 155
    estacao = Estacao(self.universo, left="250px", top="300px", width="150px", icone=PESSOA)
TypeError: __init__() got an unexpected keyword argument 'width'
'''},
{'date': 'Thu Jun 06 2019 10:22:23.931 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 167
    o_universo = Universo()
  module <module> line 155
    terra = Terra(self.universo)
  module <module> line 66
    self.terra = Elemento(TERRA, x=500, y=250, w=320, h=320, vai=self.pega)
TypeError: 'module' object is not callable
'''},
{'date': 'Tue Oct 15 2019 15:06:24.464 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 177
    o_universo = Universo()
  module <module> line 167
    estacao2 = Estacao(self.universo, left="30px", top="50px", icone=ANIMAL)
NameError: name 'ANIMAL' is not defined
'''},
{'date': 'Thu Oct 17 2019 10:21:02.966 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 186
  self.menudeanimais = Elemento(MENUDEANIMAIS[2], x=left,-40 y=top +40, w=80, h=80, vai=self.pega, cena=u)
                                                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Oct 17 2019 10:58:09.666 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: $locals_$exec_6747.Menudeanimais is not a function>
'''},
{'date': 'Thu Oct 17 2019 10:58:45.361 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: $locals_$exec_6796.Menudeanimais is not a function>
'''},
{'date': 'Thu Oct 17 2019 11:00:24.671 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 164
    o_universo = Universo()
NameError: name 'Universo' is not defined
'''},
{'date': 'Thu Oct 17 2019 11:00:55.409 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 219
    o_universo.vai()
AttributeError: 'Universo' object has no attribute 'vai'
'''},
{'date': 'Thu Oct 17 2019 11:16:29.827 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 225
    o_universo = Universo()
  module <module> line 214
    estacao3 = Estacao(self.universo, menuo, left=380, top=410, icone=OBJETO)
  module <module> line 129
    self.icone = Elemento(icone, style=dict(left=add(left,55),
  module <module> line 121
    return "{}px".format(int(valor[:-2]) + num)
AttributeError: 'int' object has no attribute '__getitem__'
'''},
{'date': 'Thu Oct 17 2019 11:28:29.672 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 190
  self.menudeobjetos = Elemento(MENUDEOBJETOS[2], x=left +50 y=top +50, w=60, h=60, vai=self.pega,cena=self.universo)
                                                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Oct 17 2019 11:28:48.63 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 190
  self.menudeobjetos = Elemento(MENUDEOBJETOS[2], x=left -50 y=top +50, w=60, h=60, vai=self.pega,cena=self.universo)
                                                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Dec 04 2020 13:16:31.325 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 233
    o_universo = Universo()
  module <module> line 216
    terra = Terra(self.universo)
  module <module> line 73
    self.terra = Elemento(TERRA, x=800, y=180, w=450, h=400, vai=self.pega)
  module elemento.main line 18
    self.img, self.title, self.dropper, self.alt = img, tit, drop, alt
  module _spy.vitollino.main line 668
    self.elt.style.backgroundImage = "url({})".format(value)
AttributeError: 'Elemento' object has no attribute 'elt'
'''},
{'date': 'Thu Jul 14 2022 10:17:59.644 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 233
    o_universo = Universo()
  module <module> line 216
    terra = Terra(self.universo)
  module <module> line 73
    self.terra = Elemento(TERRA, x=800, y=180, w=450, h=400, vai=self.pega)
  module elemento.main line 18
    self.img, self.title, self.dropper, self.alt = img, tit, drop, alt
  module _spy.vitollino.main line 668
    self.elt.style.backgroundImage = "url({})".format(value)
AttributeError: 'Elemento' object has no attribute 'elt'
'''},